tests/unit/cocoon/test_utilities.py

```diff
@@ -1711,10 +1713,16 @@ def test_strip_whitespace(self, _, val_1, val_2):
     )
     def test_create_scope_id_success(self, _, time_generator, expected_outcome):
 
-        scope_id = create_scope_id(time_generator)
+        scope_id = create_scope_id(time_generator=time_generator)
 
         self.assertEqual(first=expected_outcome, second=scope_id)
 
+    def test_create_scope_id_uuid_success(self):
+
+        scope_id = create_scope_id(use_uuid=True)
+
+        self.assertTrue(uuid.UUID(scope_id))
```

Comment:
what is this asserting? looks like its just creating a new UUID? i think  you need to assert on `scope_id` and check it 'looks' like a uuid

Reply:
if `scope_id` is not a valid uuid, `uuid.UUID` returns the Value error below

```python
Traceback (most recent call last):
  File "<input>", line 1, in <module>
  File "C:\Python38\lib\uuid.py", line 169, in __init__
    raise ValueError('badly formed hexadecimal UUID string')
ValueError: badly formed hexadecimal UUID string
```